data/filetype_extensions.conf

```diff
@@ -85,5 +86,5 @@ None=*;
 Programming=Arduino;Clojure;CUDA;Cython;Genie;Groovy;Kotlin;Nim;Scala;Swift;
 Script=Graphviz;TypeScript;
 Markup=
-Misc=JSON;
+Misc=JSON;Meson;
```

Comment:
Cmake and Makefiles are both in the scripting group, so maybe better there.

Reply:
done